kafka-connect-s3/pom.xml

```diff
@@ -141,7 +141,7 @@
                             <goal>testCompile</goal>
                         </goals>
                         <configuration>
-                            <source>1.7</source>
+                            <source>1.8</source>
```

Comment:
Was getting ```bootstrap class path not set in conjunction with -source 1.7``` when compiling even though JAVA_HOME and jenv were set to version 1.8. Hence had to make this change.